usage.log

```diff
@@ -0,0 +1,5 @@
+To build project: 
+$> mvn clean install
+
+To run test:
+$> mvn -Dtest=EspnSeleniumTest2 test
```

Comment:
It is fixed in the commit, I will populate within next a couple minutes. Especially, that CI runs tests as mvn test anyway. It also appears that it does not like EspnSeleniumTest2, so I changed the name to EspnSeleniumTest (w/o 2) to end it on "Test"